src/documents/Techniques.md

```diff
@@ -36,7 +36,7 @@ Supporting flexible styling can be as simple as using Cascading Style Sheets (CS
 
 
 Floe examples:
-* <a href="http://build.fluidproject.org/infusion/demos/uiOptions/demo.html" rel="nofollow" target="_blank" class="link-external">Learner Options</a> (sometimes known as UI Options)
+* <a href="http://build.fluidproject.org/infusion/demos/prefsFramework/" rel="nofollow" target="_blank" class="link-external">Learner Options</a> (sometimes known as UI Options)
```

Comment:
I wonder if this should actually point to the uiOptions demo, and not to the prefs framework demo...